### ISSUE TITLE
Exercise 1

### ISSUE THREAD

**Issue by atafliovich** [CSCD01/JacobChamberlain-exercise1#1]
Exercise 1 is not complete

**Comment by JacobChamberlain** [comment]
You are right, it is not complete. I was unaware that there was an exercise to do. I will try to have this completed before Monday. To avoid this issue in the future, where can I find the exercises and due dates?
